src/main/java/fruitshop/service/FileWriter.java

```diff
@@ -0,0 +1,5 @@
+package fruitshop.service;
+
+public interface FileWriter {
+    void write(String file, String fileName);
```

Comment:
The method signature for `write` in the `FileWriter` interface should include a clear parameter naming to avoid confusion. For example, it's not clear what the difference is between `file` and `fileName`. According to the checklist, avoid hardcoding and ensure that method parameters are self-explanatory. Consider renaming `file` to `content` and `fileName` to `filePath` to clarify their purposes.